json_demo/json_demo.cpp

```diff
@@ -113,6 +116,9 @@ int main() {
 
     write_addrs(file_name, addrs);
 
+    std::chrono::seconds dura(1);
+    std::this_thread::sleep_for(dura);
```

Comment:
Why the sleeps (here and below)?

Reply:
Per the commit comment: 

> json_demo.cpp, json_map_demo.cpp, json_headers_only_demo.cpp: Added "sleep()" call
>    intended to prevent unreproducible "read" failures.

I'll remove the `sleep` statements now and see if I still see those failures.